### DISCUSSION TITLE
Android with node-serialport

### DISCUSSION THREAD

**Discussion by epikao** [serialport/node-serialport#2293] [Q&A]
Hello,

I was wondering if there is any way to get node-serialport working with Android?

To create an app (apk) I use Android.js with Node.js.

What exactly needs to be changed in the code for the app to get access to the USB / Serial port from Android? Is this possible?
I read something about "Termux", "Cordova", "React", but that sounds very cumbersome....

Many thanks

**Comment by GazHank** [comment]
https://github.com/serialport/node-serialport/issues/2185 seems to be related so those users may have some answers or advice
